DakeAndDevileCorps/Screens/Store/VC/WriteReviewViewController.swift

```diff
@@ -68,3 +111,72 @@ final class WriteReviewViewController: BaseViewController {
+    
+    func imagePickerController(_ picker: UIImagePickerController, didFinishPickingMediaWithInfo info: [UIImagePickerController.InfoKey : Any]) {
+        if let image = info[UIImagePickerController.InfoKey(rawValue: "UIImagePickerControllerEditedImage")] as? UIImage {
+            reviewInputView.reviewAddPhotoView.photoList.append(contentsOf: [
+                image
+            ])
+        }
+        
+        reviewInputView.reviewAddPhotoView.photoCollectionView.reloadData()
+        picker.dismiss(animated: true, completion: nil)
+    }
+    
+    func imagePickerControllerDidCancel(_ picker: UIImagePickerController) {
+        picker.dismiss(animated: true, completion: nil)
+    }
+}
+
+extension WriteReviewViewController: UICollectionViewDataSource {
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+            return reviewInputView.reviewAddPhotoView.photoList.count
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: AddReviewPhotoCollectionViewCell.className, for: indexPath) as? AddReviewPhotoCollectionViewCell else { return UICollectionViewCell() }
+            cell.setupPhotoImageView(to: reviewInputView.reviewAddPhotoView.photoList[indexPath.row])
+            cell.deletePhotoButton.tag = indexPath.row
+            cell.deletePhotoButton.addTarget(self, action: #selector(touchUpInsideToDeletePhoto(sender:)), for: .touchUpInside)
+            return cell
+    }
+}
+
+extension WriteReviewViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        let width = (UIScreen.main.bounds.size.width - 24 * 2 - 95) / 3
+        return CGSize(width: width, height: width)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, insetForSectionAt section: Int) -> UIEdgeInsets {
+        return UIEdgeInsets(top: 0, left: (UIScreen.main.bounds.size.width - 24 * 2 - 76 * 4) / 3, bottom: 0, right:  0)
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumLineSpacingForSectionAt section: Int) -> CGFloat {
+        return 0
+    }
+
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, minimumInteritemSpacingForSectionAt section: Int) -> CGFloat {
+        return 0
+    }
+}
+
+extension WriteReviewViewController {
+    @objc func touchUpInsideToDeletePhoto(sender: UIButton) {
+        reviewInputView.reviewAddPhotoView.photoCollectionView.deleteItems(at: [IndexPath.init(row: sender.tag, section: 0)])
```

Comment:
p5;
```suggestion
        reviewInputView.reviewAddPhotoView.photoCollectionView.deleteItems(at: [IndexPath(row: sender.tag, section: 0)])
```
이렇게 써도 될 거 같아요!

Reply:
아하 굳이 초기화해줄 필요가 없군용 수정했습니다!!
[5909876](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/72/commits/590987645d94a41b758deb85e44f523bff5c641c)